.github/workflows/build-test.yml

```diff
@@ -169,7 +169,7 @@ jobs:
     needs: changes
     if: ${{ (github.event_name != 'pull_request' && !failure() && !cancelled()) || contains(fromJSON(needs.changes.outputs.samples), 'java-hello-world')}}
     with:
-      java: 17, 21
+      java: 17, 21 # Gradle Plugin supports only JDK 17+
```

Comment:
We would be able to use JDK 11 if we merge https://github.com/playframework/playframework/pull/13240

Reply:
In a next PR?

Reply:
Yes, I'll do that in next PR 😉